compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/methodgen/MainMethodGen.java

```diff
@@ -181,14 +179,6 @@ private String getJavaVersion() {
         return Objects.requireNonNullElse(javaVersion, "");
     }
 
-    private void initConfigurations(MethodVisitor mv) {
-        mv.visitVarInsn(ALOAD, 0);
-        mv.visitMethodInsn(INVOKESTATIC, JvmConstants.LAUNCH_UTILS,
```

Comment:
Can we remove the `initConfigurations()` method in the LaunchUtils with this

Reply:
We need to remove ConfigRegistry class and other relevant config classes. Will send a separate PR for this. This was removed if there is secret.txt in the user directory password will be prompt twice.